smurfs/src/components/App.js

```diff
@@ -11,7 +11,8 @@ function App(props) {
     <div className="App">
```

Comment:
add some more comments to explain your logic, this will help if your stuck, just writing what you want to be done, if you haven't already done it